osf_tests/test_generate_sitemap.py

```diff
@@ -0,0 +1,164 @@
+import os
+
+import pytest
+import mock
+import shutil, tempfile, xml, urlparse
+
+from scripts import generate_sitemap
+from osf_tests import factories
```

Comment:
I am a bit torn here between the conventions shown in osf_tests and the import guide as specified in the dev docs (http://cosdev.readthedocs.io/en/latest/style_guides/python.html#imports), which says we should import the entire module rather than a symbol in a module.